### DISCUSSION TITLE
Hi everybody!

### DISCUSSION THREAD

**Discussion by dassmod** [Cyfrin/foundry-full-course-cu#1650] [General]
Hello world!

After learning solidity for about a month in isolation I decided to commit to become a smart contract developer.
And I will be sharing all my journey on twitter. Anybody interested is welcome :) Will be posting there periodically ✌️

https://twitter.com/dassmod

**Comment by EngrPips** [comment]
Hello @dassmod, Welcome aboard. It would be interesting to see your progress.
